tests/e2e/config/jest.config.js

```diff
@@ -15,18 +17,19 @@ module.exports = {
 	preset: 'jest-puppeteer',
 
 	// Where to look for test files
-	roots: [ '<rootDir>/specs' ],
+	// roots: [ '<rootDir>/specs' ],
+	roots: [ path.resolve( __dirname, '../specs' ) ],
 
 	//setupFiles: [ '<rootDir>/.node_modules/regenerator-runtime/runtime' ],
 
 	// A list of paths to modules that run some code to configure or set up the testing framework
 	// before each test
-	setupFilesAfterEnv: [
-		'<rootDir>/jest.setup.js',
-		'<rootDir>/jest.test.failure.js',
-		'expect-puppeteer',
-	],
+	// setupFilesAfterEnv: [
+	// 	'<rootDir>jest.setup.js',
+	// 	'<rootDir>jest.test.failure.js',
+	// 	'expect-puppeteer',
+	// ],
```

Comment:
Now that the test works, maybe we can remove these comments.

Reply:
Done! Just removed all these redundant eyesores.